app/assets/stylesheets/content/_work_packages_table.lsg

```diff
@@ -3,9 +3,9 @@
 ## with work packages
 
 ```
-<div class="work-packages-table--container">
-  <div class="work-packages-table--results-container">
-    <table class="workpackages-table">
+<div class="generic-table--container work-package-table--container" interactive-table>
```

Comment:
Just a question of repetitiveness:

Do we still need the WP table component in the LSG?

Since you removed the `work-packages-table` class and this is now also a `generic-table`, I would have assumed that one example for this kind of table is enough.

You might want to use work packages as an *example* for such a table, though... (instead of the lorem ipsum)